odoo/tools/pdf.py

```diff
@@ -94,7 +96,12 @@ def add_banner(pdf_stream, text=None, logo=False, thickness=2 * cm):
     :return (BytesIO):              The modified PDF stream.
     """
 
-    old_pdf = PdfFileReader(pdf_stream, strict=False, overwriteWarnings=False)
+    try:
+        old_pdf = PdfFileReader(pdf_stream, strict=False, overwriteWarnings=False)
+        old_pdf.getNumPages()
+    except Exception:
+        raise ValidationError("Error when reading the PDF for: %s.\nIs the PDF valid ?" % text)
```

Comment:
Why `ValidationError`? An if the goal is to display the error to the end-user, why is it not translated?

Furthermore, why have questions where they're not actionable at all? The user getting the information may not even be providing the PDF. A recommendation would be more helpful

Would also be useful to have an example of the issue explained, or an OPW if one was rapported, or something.

Reply:
Thanks for the review @xmo-odoo :slightly_smiling_face: 
Unfortunately I don't have a task or ticket for this, the bug was reported within the team.
I added a copy of the traceback in the description of the PR, I hope it helps :wink: